mailer/src/main/scala/io/scalac/newspaper/mailer/db/SendOrders.scala

```diff
@@ -0,0 +1,34 @@
+package io.scalac.newspaper.mailer.db
```

Comment:
This file will be refactored. For now it just checks if we can operate on the DB and can be a copy-paste source